storage_service/administration/validators.py

```diff
@@ -0,0 +1,44 @@
+import re
```

Comment:
We're missing `from __future__ import absolute_import` at the top of this file which we need to add for later python3 compatibility. Most of the storage service modules should have this. I believe it's the only one impacted in your changes? 

This comes from testing against `pylint --py3k <module_name>`